.github/actions/github-release/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "wasmtime-github-release",
```

Comment:
Is this is an actual close on wasmtime-github-releases can we not reference it as a `@dependency` rather than cloning it?   Or will it need to be locally modified?

Reply:
I think the idea behind vendoring it in as source code was so we're not trusting external code with release assets. Additionally, we can remove a bunch of the logic around the dev branch, I think. cc @alexcrichton

Reply:
Ideally this would use things like https://github.com/actions/create-release or https://github.com/actions/upload-artifact but those didn't quite work for wasmtime's purposes.

In terms of vendoring and such it's probably easier to copy in whatever's needed here and have it evolve as this repo evolves, once you get this working it tends to not change too much anyway.

Reply:
tOK I see.  Seeing as we don't have wasmtimes requirements perhaps we can just use the off-the-shelf stuff.   Much nicer that having some bespoke JS thing (IMHO).

Per my previous suggestion how about removing the artifact uploading stuff, and have this PR simply switch from azure to github actions, then we can experiment and bikeshed the artifact upload stuff in a followup?

Reply:
Yeah, that is fine. I'll re-scope this PR to do only that.